packages/framework/src/Twig/IconExtension.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Twig;
+
+use Twig\Environment;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class IconExtension extends AbstractExtension
+{
+    /**
+     * @var \Twig\Environment
+     */
+    protected $twigEnvironment;
+
+    /**
+     * @param \Twig\Environment $twigEnvironment
+     */
+    public function __construct(
+        Environment $twigEnvironment
+    ) {
+        $this->twigEnvironment = $twigEnvironment;
+    }
+
+    /**
+     * @return \Twig\TwigFunction[]
+     */
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction('icon', [$this, 'renderIcon'], ['is_safe' => ['html']]),
+        ];
+    }
+
+    /**
+     * @param string $iconName
+     * @param array $attributes
+     * @param string|null $iconType
+     * @return string
+     */
+    public function renderIcon(string $iconName = '', array $attributes = [], ?string $iconType = 'svg-font'): string
+    {
+        $attributes['title'] = $attributes['title'] ?? '';
```

Comment:
Why use empty string as default instead of null?

Reply:
updated to null